xml/schema/types/editors.xsd

```diff
@@ -299,7 +301,71 @@
       <xs:attribute name="x" type="xs:float" use="required" />
       <xs:attribute name="y" type="xs:float" use="required" />
     </xs:complexType>
-    
+ 
+    <xs:complexType name="EditorPositionableRoundRectType">
+      <xs:annotation>
+        <xs:documentation>
+          Define the XML stucture for storing a PositionableRoundRectangle
+        </xs:documentation>
+        <xs:appinfo>
+          jmri.jmrit.display.configurexml.PositionableRoundRectXml
+        </xs:appinfo>
+      </xs:annotation>
+
+      <xs:sequence minOccurs="0" maxOccurs="unbounded">
```

Comment:
If the order of child elements does not matter in the XML file, you might want to replace xs:sequence with xs:all